sympy/functions/elementary/exponential.py

```diff
@@ -195,6 +201,23 @@ class exp(ExpBase):
     """
     The exponential function, :math:`e^x`.
 
+    Examples
+    ========
+
+    >>> from sympy.functions import exp
+    >>> from sympy.abc import x
+    >>> exp(x)
+    exp(x)
+    >>> exp(x**2 + 2*x + 1)
+    exp(x**2 + 2*x + 1)
+    >>> exp(x + 2)
+    exp(x + 2)
```

Comment:
The last two examples here don't really show much that the first doesn't already. You could show something like

```py
>>> exp(x).diff(x)
exp(x)
>>> exp(I*pi)
-1
```

Reply:
Changed.